yarn-project/Earthfile

```
@@ -144,9 +144,12 @@ test:
 
 run-e2e:
     ARG test
+    ARG debug=""
```

```suggestion
    ARG debug="aztec:*"
```
So we keep the same defaults? Or did you intend to change the default as well?

---

I did mean this actually. Now when you run with no arguments it runs all the test, but doesn't bury you in debug output.